glossary.yml

```diff
@@ -961,7 +961,7 @@
     term: "Boolean"
     def: >
       Relating to a variable or data type that can have either a logical value of
-      [true](#true) or [false](#false). Named for George Boole, a 19th
+      [true](#true) or [false](#false). Named after George Boole, a 19th
```

Comment:
The current version is not incorrect; the two usages are interchangeable in English.